extensions/google-drive/package.json

```diff
@@ -58,15 +60,20 @@
   ],
   "dependencies": {
     "@raycast/api": "^1.33.2",
+    "download": "^8.0.0",
```

Comment:
Is this package necessary? Could we download the file and then unzip with the system utililites - removes the need for `electron`?